dropbox-sdk-dotnet/Examples/SimpleTest/Program.cs

```diff
@@ -240,7 +240,8 @@ private async Task<string> GetAccessToken()
 
                     http.Start();
 
-                    System.Diagnostics.Process.Start(authorizeUri.ToString());
+                    System.Diagnostics.Process.Start(new ProcessStartInfo(authorizeUri.ToString()) { UseShellExecute = true });
+                    //System.Diagnostics.Process.Start(authorizeUri.ToString());
```

Comment:
Not sure this will work for all use cases. Please instead leave the original code, and add a comment explaining this alternative method and when you'd need to use it.